### ISSUE TITLE
"cell" class is not assigned to the cell, thus breaking the view

### ISSUE THREAD

**Issue by erKURITA** [Rudeg/react-input-calendar#39]
The `cell` class which basically brings the whole calendar together is not being assigned to the cells. I believe this is due to the warning in the example page:

```Warning: Don't set .props.classes of the React component <exports />. Instead, specify the correct value when initially creating the element or use React.cloneElement to make a new element with updated props. The element was created by exports.```

Most likely the bundle used in the example page is still using `react 0.13.x`. In `react 0.14.*` and Chrome the calendar is completely broken.

The fix is easy, though:

```javascript
/** Cell.js */
    render: function () {
        var classes = this.props.classes;
         classes += ' cell';

        return (
            React.createElement("div", {className: classes}, this.props.value)
        );
    }
```

**Comment by erKURITA** [comment]
Also, shouldn't:
```javascript
 module.exports = React.createClass({displayName: "exports", /**...*/
```
be
```javascript
 module.exports = React.createClass({displayName: "Cell", /**...*/
```
?

**Comment by erKURITA** [comment]
This appears to be fixed in the commit ce9b9f85b20fbaf6eee38fa14290e51566e40c6b

Thank you!
